README.md

```diff
@@ -1,111 +1,115 @@
-
-Emily Hood
-Washington DC
-Age 45
-Emily works at the U.S. Department of Health and Human Services (HHS) as a Director of the Office of Disease Prevention and Health Promotion. Her vast knowledge is key to this product. Due to her role, her time is limited to just one hour per week at maximum. Emily has a Software Engineer certificate from Harvard Extension and truly believes that computers and well defined algorithms helps prevent spreading disease.
-Emily will be asked for feedback and guidance for items such as: Government Health Regulations, Product feedback and probably early adopters of the Product MVP.
-
-# Product Backlog:
-Find the [Product Backlog](https://github.com/jorgecotillo/green_light_health/projects/1)
-under github projects.
-
-# Definition of Ready:
-Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
-The below checklist should be complete to ensure a story is Ready.
-- [ ] The story captures and describes a reasonable unit of work
-- [ ] The amount of work to complete the story has been estimated using the approved measurement (T-shirt Size)
-- [ ] The story is in line with the Product Vision
-- [ ] All blockers and dependencies for the needed implementation are resolved
+# Green Light Health App
+
+# Team name:
+- Coastal Vegetables
+
+# Team members:
+
+- Brendan Murphy (Scrum Master, Dev)
+- Jorge Cotillo (Dev)
+- Nicholas Owen Weber (PO, Dev)
+- Narek Asadorian (Dev)
+- Sam Negassi (Dev)
+
+# Slack channel:
+
+Join the slack channel to talk about project planning and project management.
+
+https://agilesoftwarecourse.slack.com/archives/G01632UPNAW
+
+# Github repository:
+
+All design ideas, development decisions and product vision (milestones, releases), will happen in github.
+
+# Near Vision of Product (iteration one):
+
+Green Light Health App will serve businesses and consumers to prevent the spread of illnesses. It will provide a health declaration to end-user consumers which, upon completion, businesses can use to verify their risk level before allowing them into their locations. The app will use forms of consumer contact tracing to supplement the health declarations when they may be exposed to a pathogen before showing signs of illness. 
+
+The development team's infrastructure will include scripted environment creation, automated testing, continuous integration, and continous delivery for all components. This will expedite the delivery rate of product increments and increase the efficiency of the development team to help further this product's cause.
```

Comment:
Is this type of thing supposed to be part of the vision? Does it not leak infrastructure details into otherwise product-focused copy?